hazelcast/src/test/java/com/hazelcast/client/config/XmlClientConfigBuilderTest.java

```diff
@@ -665,6 +668,20 @@ public void testPersistentMemoryConfiguration_NotExistingModeThrows() {
         buildConfig(xml);
     }
 
+    @Test
+    @Override
+    public void testNativeMemoryConfiguration_isBackwardCompatible() {
```

Comment:
Should we also change the `nativeMemory` test in this file to use the new capacity field, instead of size?

Reply:
1135d7cd77ef1592681bdc17a0779a1f4d21915e